continuous_integration/generate_job_matrix.py

```
@@ -102,12 +102,20 @@
     pio_to_acli = json.load(f)
 
 # Find all of the non-menu examples
-non_menu_examples = [
-    f
-    for f in os.listdir(examples_path)
-    if os.path.isdir(os.path.join(examples_path, f))
-    and f not in [".history", "logger_test", "archive", "tests", menu_example_name]
-]
+non_menu_examples = []
+for root, subdirs, files in os.walk(examples_path):
```

Ruff (0.15.1)

[warning] Loop control variable `subdirs` not used within loop body

Rename unused `subdirs` to `_subdirs`

(B007)